src/ga4gh/vrs/extras/translator.py

```diff
@@ -366,105 +286,36 @@ def _from_gnomad(self, gnomad_expr, **kwargs):
         g = m.groupdict()
         assembly_name = kwargs.get("assembly_name", self.default_assembly_name)
         sequence = assembly_name + ":" + g["chr"]
-        refget_accession = self._get_refget_accession(sequence)
+        refget_accession = self.data_proxy.derive_refget_accession(sequence)
         if not refget_accession:
             return None
+        
+        print("refget_accession: ", refget_accession)
 
         start = int(g["pos"]) - 1
         ref = g["ref"].upper()
         alt = g["alt"].upper()
         end = start + len(ref)
         ins_seq = alt
 
-        # validation checks
-        valid_ref_seq, err_msg = self._is_valid_ref_seq(sequence, start, end, ref)
-        if kwargs.get("require_validation", True) and not valid_ref_seq:
-            raise ValidationError(err_msg)
-
-        seq_ref = models.SequenceReference(refgetAccession=refget_accession)
-        location = models.SequenceLocation(sequenceReference=seq_ref, start=start, end=end)
-        sstate = models.LiteralSequenceExpression(sequence=ins_seq)
-        allele = models.Allele(location=location, state=sstate)
-        allele = self._post_process_imported_allele(allele, **kwargs)
+        # TODO: ask other devs if this should be down on all _from_...  methods?
```

Comment:
I have a PR (#375) that simplifies this. I can update it once this PR gets merged.
```suggestion
```

Reply:
+1